picker/src/main/java/me/angrybyte/numberpicker/view/ActualNumberPicker.java

```diff
@@ -616,13 +616,13 @@ private void onControlClicked(@Control int which) {
             case FAST_ARR_LEFT: {
                 int valueChange = (mMaxValue - mMinValue) / 10;
                 mValue -= valueChange;
-                changeX = (int) -(0.1f * mWidth);
```

Comment:
This hard-codes visual movement to literally 1px, which is probably invisible to the naked eye on high-density displays.